Source/Extensions/Blazorise.DataGrid/_DataGridCellSelectEdit.razor.cs

```diff
@@ -73,6 +74,11 @@ private void OnSelectedValueChanged( object value )
         }
     }
 
+    private async Task OnSelectedValuesChanged( IReadOnlyList<object> values )
+    {
+        await CellValueChanged.InvokeAsync( values?.Select( x => x.ToString() )?.ToArray() );
```

Comment:
**Reminder to self :** Can we somehow support more types other then string[]? int[], so on...